snap/info.go

```diff
@@ -929,3 +929,12 @@ func JoinSnapApp(snap, app string) string {
 	}
 	return fmt.Sprintf("%s.%s", snap, app)
 }
+
+// Nickname returns the nickname for given snap name. If there is none, returns
+// the original name.
+func Nickname(snapName string) string {
```

Comment:
Isn't this the same function we have above?  Can we use the same without cycles?

Also, perhaps similar to the above point:

- snap.UseNick
- snap.DropNick